Library/ViewModels/CommentsViewModel.swift

```diff
@@ -22,15 +22,15 @@ public protocol CommentsViewModelInputs {
 }
 
 public protocol CommentsViewModelOutputs {
-  /// Emits a URL for the avatar image view.
-  var avatarURL: Signal<URL?, Never> { get }
+  /// Emits a CommentComposerViewData object that determines the avatarURL and whether the user is a backer.
+  var configureCommentComposerViewWithData: Signal<CommentComposerViewData, Never> { get }
+
+  /// Emits a boolean that determines if the comment input area is visible.
+  var isCommentComposerHidden: Signal<Bool, Never> { get }
```

Comment:
`commentComposerIsHidden` may be more consistent with our other _hidden_ outputs.

Reply:
I think we only have this one and `isCommentsLoading` so both correspond. I don't mind changing both but they look like they're matching?